rollup.prod.config.js

```diff
@@ -33,6 +36,12 @@ export default {
     resolve(),
     commonjs(),
     babel(),
+    replace({
+      delimiters: ["{{", "}}"],
+      values: {
+        version
+      }
+    }),
```

Comment:
I decided to only swap `{{version}}` with the real package version when creating a production build. My reasoning was that it will be less code to maintain and it doesn't seem valuable/applicable in non-prod builds.